app/models/pawn.rb

```diff
@@ -1,4 +1,24 @@
 class Pawn < Piece
+
+	#en passant status of self
+	def update_en_passant_status(end_x, end_y)
+		if (self.pos_y - end_y).abs == 2
+			self.update_attributes(en_passant: true)
+		else
+			self.update_attributes(en_passant: false)
+		end
+	end
+
+	#determine if pawn can capture opposing pawn en passant
+	def can_capture_en_passant(opponent_pawn)
+		if opponent_pawn.updated_at == game.updated_at && opponent_pawn.en_passant == true 
+			return true
```

Comment:
Maybe you can track the time of the last piece moved for the white en_passant pawn. And then cycle through the white pieces to see if the most recent white piece update is newer than the white en_passant pawn. Then if it's newer set it back to false?